include/openssl/crypto.h

```diff
@@ -217,6 +217,12 @@ OPENSSL_EXPORT void OPENSSL_cleanup(void);
 // |BORINGSSL_FIPS| and zero otherwise.
 OPENSSL_EXPORT int FIPS_mode_set(int on);
 
+// |CRYPTO_mem_ctrl| intentionally does nothing and returns 0.
```

Comment:
CI is failing on the documentation check. 
```suggestion
// CRYPTO_mem_ctrl intentionally does nothing and returns 0.
```